plugins/auth-api/package.json

```
@@ -19,17 +19,19 @@
     "typescript": "4.6.2"
   },
   "dependencies": {
-    "@amplication/code-gen-types": "^1.1.11",
-    "@amplication/code-gen-utils": "^0.0.2",
+    "@amplication/code-gen-types": "1.1.20",
+    "@amplication/code-gen-utils": "^0.0.3",
     "@babel/parser": "^7.18.11",
     "@babel/types": "^7.18.10",
+    "@extra-set/difference": "^2.2.4",
     "@nestjs/common": "^9.0.9",
     "@nestjs/core": "^9.0.9",
     "@nestjs/jwt": "^9.0.0",
```

Please use the version that is not affected by `jsonwebtoken` vulnerability
```suggestion
    "@nestjs/jwt": "^10.0.1",
```